src/app/acesso/cadastro/info-pessoal-cadastro/info-pessoal-cadastro.component.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { InfoPessoalCadastroComponent } from './info-pessoal-cadastro.component';
+import { MatListModule } from '@angular/material/list';
+import { MatInputModule } from '@angular/material/input';
+import { MatSelectModule } from '@angular/material/select';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { LoggerTestingModule } from 'ngx-logger/testing';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { RouterTestingModule } from '@angular/router/testing';
+import { TranslateModule } from '@ngx-translate/core';
+import { MatDialogModule } from '@angular/material/dialog';
+import { MatCardModule } from '@angular/material/card';
+import { ChangeDetectorRef } from '@angular/core';
+
+describe('InfoPessoalCadastroComponent', () => {
+  let component: InfoPessoalCadastroComponent;
+  let fixture: ComponentFixture<InfoPessoalCadastroComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ InfoPessoalCadastroComponent ],
+      providers: [
+        ChangeDetectorRef
+      ],
+      imports: [
+        MatListModule,
+        MatInputModule,
+        MatSelectModule,
+        BrowserAnimationsModule,
+        HttpClientTestingModule,
+        LoggerTestingModule,
+        FormsModule,
+        RouterTestingModule,
+        ReactiveFormsModule,
+        TranslateModule.forRoot(),
```

Comment:
Codacy found an issue: ['beforeEach' is not defined.](https://app.codacy.com/manual/gaugsouza/petshow-frontend/pullRequest?prid=6334730)